dapp/src/components/CouponInputSelection.tsx

```diff
@@ -42,19 +42,18 @@ export function CouponInputSelection({ coupons, onAddCoupon }: CouponInputSelect
 
             <div className="flex flex-col items-start gap-y-sm">
                 <Input
-                    placeholder={
-                        coupons.length === 0 ? 'Enter a coupon code' : 'Add an extra coupon'
-                    }
+                    placeholder={'Have a discount code?'}
                     type={InputType.Text}
                     value={coupon}
                     onChange={(e) => setCoupon(e.target.value)}
                     onKeyDown={(e) => e.key === 'Enter' && addCoupon()}
                     onClearInput={() => setCoupon('')}
+                    disabled={coupons.length > 0}
                 />
                 <ButtonUnstyled
                     className="bg-names-gradient-primary bg-clip-text text-transparent bg-[length:200%] enabled:transition-[background-position] enabled:duration-500 enabled:hover:bg-[100%] text-label-md cursor-pointer disabled:opacity-40 disabled:cursor-not-allowed"
                     onClick={addCoupon}
-                    disabled={!coupon.trim()}
+                    disabled={coupons.length > 0}
```

Comment:
tbd: do we need to have it disabled or we can don't render field?

<img width="485" height="288" alt="image" src="https://github.com/user-attachments/assets/2ea90090-e8ab-4ec3-b7b6-925b03c99faf" />
<img width="487" height="394" alt="image" src="https://github.com/user-attachments/assets/7ca48dd3-6e2d-4b0e-955e-7b8b878698b5" />